scripts/generate_rust_analyzer.py

```diff
@@ -71,11 +71,19 @@ def append_crate(display_name, root_module, is_workspace_member, deps, cfg):
     )
     crates[-1]["proc_macro_dylib_path"] = "rust/libmodule.so"
 
+    append_crate(
+        "link_time_panic",
+        srctree / "rust" / "link_time_panic.rs",
+        True,
+        ["core", "compiler_builtins"],
+        [],
+    )
+
     append_crate(
         "kernel",
         srctree / "rust" / "kernel" / "lib.rs",
         True,
-        ["core", "alloc", "module"],
+        ["core", "alloc", "link_time_panic", "module"],
```

Comment:
Could you do this in order of the declaration above? So either move the append_crate for `link_time_panic` above `module` or switch it here.